src/shared/watchdog.c

```diff
@@ -52,7 +53,7 @@ static int get_watchdog_sysfs_path(const char *filename, char **ret_path) {
         if (!S_ISCHR(st.st_mode))
                 return -EBADF;
 
-        if (asprintf(ret_path, "/sys/dev/char/%u:%u/%s", major(st.st_rdev), minor(st.st_rdev), filename) < 0)
+        if (asprintf(ret_path, "/sys/dev/char/"DEVNUM_FORMAT_STR"/%s", DEVNUM_FORMAT_VAL(st.st_rdev), filename) < 0)
```

Comment:
doesn't matter, but more often than not we'd put spaces between the `"` and the `DEVNUM_FORMAT_STR` and the `"`